packages/gatsby-telemetry/src/index.js

```diff
@@ -7,7 +7,10 @@ process.on(`exit`, flush)
 
 // For longrunning commands we want to occasinally flush the data
 // The data is also sent on exit.
-const interval = 10 * 60 * 1000 // 10 min
+
+const interval = process.env.TELEMETRY_BUFFER_INTERVAL
+  ? Number(process.env.TELEMETRY_BUFFER_INTERVAL)
```

Comment:
```suggestion
  ? Math.max(Number(process.env.TELEMETRY_BUFFER_INTERVAL), 1000)
```

just sanity clamp to not spawn flush process too often if wrongly configured - 1 second is my suggestion, feel free to adjust

Reply:
Great idea for a safeguard, thanks!